#!/usr/bin/env python
PACKAGE = "perception_pc_segmentation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_inliers",    int_t,    0, "Detection minimum inliers", 15,  6, 100)
gen.add("min_inliers_redetect",    int_t,    0, "Detection minimum inliers redetection", 35,  6, 100)
gen.add("debug",    bool_t,    0, "Print debug messages", True)
gen.add("use_previous_detection",    bool_t,    0, "Use the pose of the previous detection for the next frame", True)

exit(gen.generate(PACKAGE, "perception_pc_segmentation", "Feature"))
